Permanent/ViewControllers/LegacyPlanning/ViewController/LegacyPlanningLoadingViewController.swift

```diff
@@ -16,11 +16,18 @@ class LegacyPlanningLoadingViewController: BaseViewController<LegacyPlanningView
         
         navigationItem.title = "Legacy Planning".localized()
         
-        if viewModel?.getLegacyPlanningAccount() == true {
-            showStatus()
-        } else {
-            showIntro()
-        }
+        viewModel?.getLegacyPlanningAccount(completion: { [weak self] result in
+            switch result {
+            case .success(let haveLegacySteward):
+                if let haveLegacySteward = haveLegacySteward, haveLegacySteward {
```

Comment:
if haveLegacySteward != nil would be easier as you don't use the haveLegacySteward. Also, is this a bool or an object? If it's an object you should name it something else like: steward